File: answers/leetcode/705. Design HashSet/Solution.py
class MyHashSet:

    def __init__(self):
        self.exist_list = [False] * 1000001

    def add(self, key: int) -> None:
        self.exist_list[key] = True

    def remove(self, key: int) -> None:
        self.exist_list[key] = False

    def contains(self, key: int) -> bool:
        return self.exist_list[key]


# Your MyHashSet object will be instantiated and called as such:
# obj = MyHashSet()
# obj.add(key)
# obj.remove(key)
# param_3 = obj.contains(key)

File: answers/leetcode/705. Design HashSet/Solution.java
class MyHashSet {

    private static final int SIZE = 1000001;

    private boolean[] array = new boolean[SIZE];

    /** Initialize your data structure here. */
    public MyHashSet() {
    }
    
    public void add(int key) {
        this.array[key] = true;
    }
    
    public void remove(int key) {
        this.array[key] = false;
    }
    
    /** Returns true if this set contains the specified element */
    public boolean contains(int key) {
        return this.array[key];
    }
}

/**
 * Your MyHashSet object will be instantiated and called as such:
 * MyHashSet obj = new MyHashSet();
 * obj.add(key);
 * obj.remove(key);
 * boolean param_3 = obj.contains(key);
 */